src/DynamoCore/Search/SearchElements/SearchElementBase.cs

```diff
@@ -43,7 +43,7 @@ public abstract class SearchElementBase : BrowserInternalElement
         /// Higher = closer to the top of search results </value>
         public abstract double Weight { get; set; }
 
-        public virtual void Execute()
+        public new virtual void Execute()
```

Comment:
ugh ...not sure about the implications here.
SearchElementBase inherits Execute from a base class but, in the base class it is declared as internal. However SearchElementBase declares it as public.
It should have just been
```
internal override void Execute()
```

Reply:
that way works, but I 'm not sure whether this method is being exposed to other assemblies, so I think the easiest way is to define a new member with the required visibility.